app/src/main/java/assembly/giraff/andtinder/view/CardContainer.java

```diff
@@ -401,6 +400,14 @@ public void setGravity(int gravity) {
 		mGravity = gravity;
 	}
 
+    // TODO is it the correct way to get currently shown card?
```

Comment:
@alouanemed  I think @akhyrul is not sure how to get the currently shown card (when you tap to share).